Default (Linux).sublime-keymap

```diff
@@ -51,6 +51,50 @@
         "command": "cscope_visiter",
         "context": [{"key": "selector", "operand": "text.find-in-files"}]
     },
+    {
+        "keys": ["j"],
```

Comment:
I think I'd rather leave out the sublime-keymap changes at least for the first round and just make the changes that are required to support it. Can you remove these sublime-keymap changes for at least this first review and let's get through all the other changes first?

Reply:
Removed